src/Extractor.ts

```diff
@@ -1070,24 +1137,31 @@ class Extractor {
       // This is fine, we just don't know what it is. This should be the expected
       // case for operation types such as `Query`, `Mutation`, and `Subscription`
       // where there is not strong convention around.
+    } else if (
+      node.type.kind === ts.SyntaxKind.TypeReference &&
```

Comment:
What's the reason we can't also support external interfaces? I think you could define an interface which extends the interface from the other library.